cuizinart/frontend/src/components/dataSetsComp.jsx

```diff
@@ -29,11 +29,34 @@ export default class SideBar extends Component {
                             {d.value}
                         </div>
                         <div className="card-body">
+                            <h5><span className="label label-default">Variables: </span></h5>
                             {
                                 d.vars.map( va =>
                                     <div key={`div-${shortid.generate()}`} className="form-check">
                                         <label className="form-check-label">
-                                            <input type="checkbox" className="form-check-input" checked={va.selected} onChange={(e) => this.handleCheckbox(va.key, e)}/>{va["key"]}: {va["description"]}
+                                            <input type="checkbox" className="form-check-input" checked={va.selected} onChange={(e) => this.handleCheckbox('vars', va.key, e)}/>{va["key"]}: {va["description"]}
+                                        </label>
+                                    </div>
+                                )
+                            }
+                            <h5 className="mt-3"><span className="label label-default">Forecast Windows: </span></h5>
+                            {
+                                d.horizons.map( va =>
+                                    <div key={`div-${shortid.generate()}`} className="form-check form-check-inline">
+                                        <label className="form-check-label">
+                                            <input type="checkbox" className="form-check-input" checked={va.selected} onChange={(e) => this.handleCheckbox('horizons', va.key, e)}/>
+                                            <label className="form-check-label" >{va["key"]}</label>
+                                        </label>
+                                    </div>
+                                )
+                            }
+                            <h5 className="mt-3"><span className="label label-default">Forecast Issues: </span></h5>
+                            {
+                                d.issues.map( va =>
+                                    <div key={`div-${shortid.generate()}`} className="form-check form-check-inline">
+                                        <label className="form-check-label">
+                                            <input type="checkbox" className="form-check-input" checked={va.selected} onChange={(e) => this.handleCheckbox('issues', va.key, e)}/>
```

Comment:
Showing the time down to seconds seems a bit overkill (and waste of space) to me. Can we change it to HH:MM?